src/js/components/reviewData/ReviewDataContent.jsx

```diff
@@ -175,15 +175,21 @@ export default class ReviewDataContent extends React.Component {
 
         let certifyButtonText = "You do not have permissions to certify";
         let notifyButtonText = "Notify Another User that the Submission is Ready for Certification";
-        let buttonClass = "btn-disabled";
+        let buttonClass = " btn-disabled";
         let buttonAction = "";
         let monthlySubmissionError = null;
-        if (this.props.data.publish_status == "published") {
+
+        if(this.props.data.gtas && this.props.data.gtas) {
```

Comment:
Do we want to display this over "already certified" and "monthly cannot be certified"? I'd imagine we'd want to display those first, then "certification not allowed during GTAS" only if neither of those is true. I think those messages are more important